src/ol/render/canvas/TextBuilder.js

```diff
@@ -429,7 +436,7 @@ class CanvasTextBuilder extends CanvasBuilder {
         this.text_,
         this.textKey_,
         this.strokeKey_,
-        this.fillKey_,
+        this.fillKey_ ? defaultFillKey : this.fillKey_,
```

Comment:
This should be either the `defaultFillKey` or `null`, right? The above can also result in `undefined`. I don't think this is desired.

Reply:
falsy should always be an empty string

Reply:
Can you point to a test or a section of code that backs this?

Reply:
There are only two places in the module where `this.fillKey_` is set.  It is always a string.  `fillKey` is copied from `this.fillKey_`.  In any case the executor only checks for truthy/falsy.